packages/atomic/src/components/search/result-template-components/atomic-result-image/atomic-result-image.tsx

```diff
@@ -27,6 +27,13 @@ export class AtomicResultImage implements InitializableComponent {
    */
   @Prop({reflect: true}) field!: string;
 
+  /**
+   * The result field that contains the alt text for the image. This will look for the field in the Result object first, then in the Result.raw object
+   *
+   * If the field is not specified, or does not contain a valid value, the alt text will be set to "Image for {productName}".
+   */
+  @Prop({reflect: true}) imageAltField?: string;
```

Comment:
Wondering if I should rename `imagesAltField` to `imageAltField` on atomic-product-image for consistency between these two components

Reply:
It would be a breaking change because the attribute is part of the public API.